generation/spade_gan/spade_gan.ipynb

```diff
@@ -0,0 +1,991 @@
+{
```

Comment:
<p>Should be the fourth reference here instead of the fifth.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>